drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -4061,6 +4061,10 @@ RID RenderingDeviceVulkan::shader_create(const Vector<ShaderStageData> &p_stages
 							layout_binding.descriptorType = VK_DESCRIPTOR_TYPE_INPUT_ATTACHMENT;
 							info.type = UNIFORM_TYPE_INPUT_ATTACHMENT;
 						} break;
+						case SPV_REFLECT_DESCRIPTOR_TYPE_ACCELERATION_STRUCTURE_KHR: {
+							ERR_PRINT("Acceleration structure not supported.");
+							continue;
+						} break;
```

Comment:
FYI, this is new in this update. Had to handle it to fix a compilation warning, but since I don't know if we have a use for it, I marked it as unsupported and skipped it (as done for a couple other enum values above).